README.md

```diff
@@ -6,15 +6,60 @@
 
 ## Overview
 
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+This is a generated connector for [Alfresco](https://api-explorer.alfresco.com/api-explorer/) OpenAPI specification. 
+
+This Alfresco Content Services REST APIs enable you to access to the core features of Alfresco Content Services.
```

Comment:
Let's follow the wording of the other connectors here. Something like

[Alfresco] is a <what it does>. 

The `ballerinax/alfresco` connectors offers APIs to connect and interact with the [Alfresco endpoints](Link to documentation), specifically based on the [Alfresco REST API Version x](API reference)